File: app/src/main/java/com/example/studyit/Fragment_notes.java
package com.example.studyit;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.studyit.R;

import androidx.fragment.app.Fragment;

public class Fragment_notes extends Fragment {
    public Fragment_notes() {
// Required empty public constructor
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
// Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_notes, container, false);
    }



}

File: app/src/main/java/ru/psu/studyit/view/activities/lab/Fragment_notes.kt
//@author Дегтяникова Дарья
package ru.psu.studyit.view.activities.lab

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import ru.psu.studyit.R

import androidx.fragment.app.Fragment

class Fragment_notes : Fragment()
{

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View?
    {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_lab_comments, container, false)
    }


}// Required empty public constructor
